kda-env/init-kadena.repl

```diff
@@ -3,6 +3,7 @@
 (env-sigs [])
 
 ; Root namespace
+(load "./kadena/ns.pact")
```

Comment:
Not sure how you want to deal with this addition — presumably we also ought to add a `(create-table registry)` call so it all works.